tests/kallax.go

```diff
@@ -86,6 +86,9 @@ func NewEventsFixtureStore(db *sql.DB) *EventsFixtureStore {
 // required for this operation.
 func (s *EventsFixtureStore) Insert(record *EventsFixture) error {
 
+	if err := record.BeforeInsert(); err != nil {
```

Comment:
yes yes, I got it once I started fixing it.
@ajnavarro @erizocosmico done at 3bd4a44